test/test_unary_ufuncs.py

```diff
@@ -33,6 +202,190 @@ def test_unsupported_dtypes(self, device, dtype, op):
         with self.assertRaises(RuntimeError):
             op(t)
 
+    # Tests bool tensor negation raises the correct error
+    def test_neg_error_message(self, device):
+        self.assertRaisesRegex(
+            RuntimeError,
+            r"Negation, the `\-` operator, on a bool tensor is not supported. "
```

Comment:
If you're going to match for this much of the error message, it would be better to just use an expect test (assertExpectedInline) so it is easier for people to update the test if they modify the error message.

Reply:
Sure. This was just copied from the existing neg test.

Reply:
Oh, this is preexisting code

Reply:
Might as well fix it while it's being copy-pasted. I made it more maintainable.